packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -72,13 +73,14 @@ export class SearchBoxBase extends BaseComponent<ISearchBoxProps, ISearchBoxStat
       hasFocus,
       disabled,
       hasInput: value!.length > 0,
-      disableAnimation
+      disableAnimation,
+      iconClassName: iconProps ? iconProps.className : undefined
```

Comment:
My only curiosity is if we should be adding this to the StyleProps. I understand the reasoning, but if we do this one, what about the close button className, or all of the other possible classNames in other components. You can always pass in custom classNames via styles={{icon:'classname'}}

Reply:
yeah, after chatting a bit, let's not pass the iconClassName down right now. it'll clutter up the styleprops interface, and the prescribed way to pass down classNames is through styles (which already has all of the proper endpoints).

Other than that, I'm 100% good with this.

Reply:
Yeah no worries, I'm by no means familiar enough with this project to know what the idiomatic aproach is. I also noticed this wasn't using `subComponentStyles` but I'm not sure when/why that's used either.

Reply:
yeah, when I did this conversion the subComponents were available yet. At some point we need to go back and add that in where we can, though many of those subComponents are buttons....and you know first hand we still have issues there :D